timer/__init__.py

```diff
@@ -12,20 +12,26 @@
 from albertv0 import *
 from threading import Timer
 from time import strftime, time, localtime
+import dbus
 import os
+import random
 import subprocess
 
 __iid__ = "PythonInterface/v0.1"
 __prettyname__ = "Timer"
-__version__ = "1.1"
+__version__ = "1.2"
 __trigger__ = "timer "
 __author__ = "Manuel Schneider, Andreas Preikschat"
```

Comment:
Please use your github username for easy mentions. When you're at it please change mine to manuelschneid3r as well.

Reply:
done